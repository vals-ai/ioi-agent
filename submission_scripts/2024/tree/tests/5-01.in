version https://git-lfs.github.com/spec/v1
oid sha256:7b647da5df77849941bbded3131c114eb8dfa3c00cae0cc1a0b0b4dfed2dad73
size 3038431
