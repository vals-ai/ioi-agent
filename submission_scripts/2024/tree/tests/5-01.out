version https://git-lfs.github.com/spec/v1
oid sha256:7c6cd35094ecfea830e86e256433dd397ba4e2664de320b4571d0b283aa4381b
size 1165498
